contributor-docs/operations/tests/test-implementation-guide.md

```diff
@@ -0,0 +1,147 @@
+---
+title: SCS Conformance Test Implementation Guide
+type:
+status: Draft
+track: Global
+---
+
+SovereignCloudStack (SCS) uses [conformance tests][gh-scripts] to certify
+compliance of a given cloud offering with respect to a given [certificate
+scope][cert-scopes] such as *SCS Compatible IaaS v4*.
+Our aim is that these tests are reliable, consistent and comprehensible for
+the users.
+This document is a guideline for conformance test authors and summarizes the
+current best practices to achieve this goal.
+
+## 1. Mapping of RFC2119 Keywords
+
+Test authors working on new and existing conformance tests for an SCS standard
+must implement the keywords such as MUST and SHOULD according to the [SCS
+RFC2119 Keyword Test Guide][scs-rfc2119-guide].
+
+## 2. Unit and Regression Tests
+
+Test authors are *strongly* encouraged to include unit and regression tests for
+the conformance test's logic.
+Conformance tests will inevitably contain some non-trivial algorithms, be it for
+parsing flavor names or evaluating CVE vulnerability reports.
+As a bonus, a well written unit test also makes it easier for reviewers to
+cross check a given pull request with new or enhanced conformance tests.
```

Comment:
This is a non sequitur. I think a linking sentence is missing. The sentence before only states a problem without saying why having a unit test is good (which is kind of self evident whn you think about it, but for the flow of the text, it should be added).